tests/python_tests/samples/conftest.py

```diff
@@ -207,6 +213,36 @@ def download_test_content(request):
             os.remove(file_path)
 
 
+@pytest.fixture(scope="session")
+def generate_test_content(request):
+    """Download the test content from the given URL and return the file path."""
```

Comment:
```suggestion
    """Generate an image of lines and return the file path."""
```
Maybe it's even worth renaming the function because it can only generate a specific image

Reply:
The docstring didn't change

Reply:
oh, I've changed it for download_test_content ((